pkg/handler/data_recorder_pubsub.go

```diff
@@ -0,0 +1,101 @@
+package handler
+
+import (
+	"context"
+	"encoding/json"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/checkr/flagr/pkg/config"
+	"github.com/checkr/flagr/swagger_gen/models"
+	"google.golang.org/api/option"
+
+	"github.com/sirupsen/logrus"
+)
+
+type pubsubRecorder struct {
+	enabled  bool
```

Comment:
is this `enabled` necessary here? eval.go checks it here https://github.com/checkr/flagr/blob/master/pkg/handler/eval.go#L178

Reply:
oh yeah... this could probably be removed from the kafka and kinesis versions as they do the check as well

Reply:
didn't realize that it was also used in kafka and kinesis, we can probably clean them up in other PRs